test/FakeServer.js

```diff
@@ -18,7 +19,7 @@ FakeServer.prototype.start = function (cb) {
     this._socket = dgram.createSocket('udp4');
 
     this._socket.on('message', function (msg, rinfo) {
-        //console.warn("Server got: '" + msg.toString() + "'");
+        debug("Server got: '" + msg.toString() + "'");
```

Comment:
Huh? I'm using `require('util').debuglog('fakeserver')`. Behaves the same as the good 'ol `debug`-package.